crates/ruff/src/rules/flake8_pyi/snapshots/ruff__rules__flake8_pyi__tests__PYI018_PYI018.pyi.snap

```diff
@@ -0,0 +1,22 @@
+---
+source: crates/ruff/src/rules/flake8_pyi/mod.rs
+---
+PYI018.pyi:4:1: PYI018 TypeVar `_T` is never used
+  |
+2 | from typing import TypeVar
+3 | 
+4 | _T = typing.TypeVar("_T")
+  | ^^ PYI018
```

Comment:
Should we highlight the entire declaration?

Reply:
I personally think highlighting just the symbol is ok.